apps/product/tasks.py

```
@@ -1,207 +1,1075 @@
+def not_official_api_wildberries(nmId, api_key):
+    
+    response = requests.get(f"https://card.wb.ru/cards/detail?dest=-446085&regions=80,83,38,4,64,33,68,70,30,40,86,75,69,1,66,110,22,48,31,71,112,114&nm={nmId}")
+    result = []
+    if response.status_code == 200:
+        products = response.json()['data']['products']
+        for item in products:
+            for item2 in item["sizes"]:
+                for item3 in item2['stocks']:
+                    warehouse_id = item3['wh']
+                    warhouse_name = get_warehouse_name_wildberries(warehouse_id=warehouse_id,api_key=api_key)
+                    if type(warhouse_name) != bool:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)